src/App.css

```diff
@@ -0,0 +1,27 @@
+@import url("./Components/Navigation/navigation.css");
+@import url("./Components/Toast/toast.css");
+@import url("./Components/Video/videoCard.css");
+@import url("./Components/Playlist/playlist.css");
+@import url("./Components/VideoPlayer/videoPage.css");
+@import url("./Components/Video/filters.css");
+@import url("./Components/Video/videoList.css");
+@import url("./Components/History/history.css");
```

Comment:
the css files are placed in the respective component's folder itself. However, the reason to import them on app.css is so that other components could also use these styles.